packages/buidler-core/test/internal/solidity/resolver.ts

```diff
@@ -486,6 +486,24 @@ describe("Resolver", function () {
           ERRORS.RESOLVER.IMPORTED_FILE_NOT_FOUND
         );
       });
+
+      it("Should let you import a library file with its relative path from a local file", async function () {
+        try {
```

Comment:
Is this `try/catch` necessary?

Reply:
Oops, it's a hack I use to see the full chained stack trace in failed tests 😅

Reply:
Removed it